.github/workflows/lint.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint Code
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
```

Comment:
nit: this should be `lint` (not `build`)

Reply:
RAPID ITERATION